gulpfile.js

```diff
@@ -65,6 +100,34 @@ gulp.task('process-app-js', () => {
 	});
 });
 
+
+gulp.task('process-featured-events-widget-js', () => {
```

Comment:
I see what you're doing here, but I think if we're going to have feature-specific sections, it might be a better idea to have a `/gulp-tasks/` folder (or something like that) with objects you can export and include in the gulpfile. Maybe your object is just the list of files, and it gets imported and added to the `gulp.src` list?

Reply:
Something like:
```JavaScript
const files = [
    'js/apps/events-page/app.js',
    'js/apps/events-page/featuredEvents*.js',
    '/apps/events-page/constants.js',
    'js/apps/events-page/config.js',
    'js/apps/events-page/dataServices/**/*.js',
    'js/apps/events-page/filters/**/*.js',
    'js/apps/events-page/services/**/*.js',
    'js/apps/events-page/controllers/**/featuredEvents*.js',
    'js/apps/events-page/directives/**/featuredEvents.js'];

module.exports = files;
```

And then you can just import it and add it to the `process-master-js` task's file list.

Reply:
done